pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/content/I18nContentRenderer.java

```diff
@@ -100,6 +106,14 @@ public static RenderValueProvider.Builder newValueProviderBuilder(Handle handle,
         builder.setParticipantTimeZone(zone);
         builder.setDate(LocalDate.now(zone));
 
+        builder.setActivityInstanceCreationDate(handle.attach(ActivityInstanceDao.class)
+                .findBaseResponseByInstanceId(activityInstanceId)
+                .map(ActivityResponse::getFirstCompletedAt)
+                .map(Instant::ofEpochMilli)
+                .map(x -> x.atZone(ZoneId.systemDefault()))
```

Comment:
There's a resolved ZoneId for the participant above- is there a reason that being ignored and the system's default zone is being used instead?

Reply:
Ah, I see what's going on here. 👍